content/learn/quick-start/getting-started/setup.md

```diff
@@ -164,6 +164,8 @@ cargo add bevy -F dynamic_linking
 
 {% callout(type="warning") %}
 On Windows you must also enable the [performance optimizations](#compile-with-performance-optimizations) or you will get a ["too many exported symbols"](https://github.com/bevyengine/bevy/issues/1110#issuecomment-1312926923) error.
+
+In order to run `cargo test --doc`, you must also add `%USERPROFILE%\.rustup\toolchains\stable-x86_64-pc-windows-msvc\lib\rustlib\x86_64-pc-windows-msvc\lib\ ` to your `PATH` environment variable. Replace the part saying `stable` with `nightly` if you're running the nightly compiler.
```

Comment:
You can also use `rustc --print target-libdir` to print the lib directory (so that the user doesn't have to manually adjust the path).

Reply:
Didn't know about that, thanks :) [`924dae3` (#1522)](https://github.com/bevyengine/bevy-website/pull/1522/commits/924dae3a0dcde8e87fb0524e87eb05203462d057)